zerver/views/webhooks/trello/card_actions.py

```diff
@@ -0,0 +1,197 @@
+from typing import Dict, Tuple, Any
+from datetime import datetime
+from .exceptions import UnknownUpdateCardAction
+from .templates import TRELLO_SUBJECT_TEMPLATE, TRELLO_MESSAGE_TEMPLATE
+
+SUPPORTED_CARD_ACTIONS = [
+    'updateCard',
+    'createCard',
+    'addLabelToCard',
+    'removeLabelFromCard',
+    'addMemberToCard',
+    'removeMemberFromCard',
+    'addAttachmentToCard',
+    'addChecklistToCard',
+    'commentCard'
+]
+
+CREATE = 'createCard'
+CHANGE_LIST = 'changeList'
+CHANGE_NAME = 'changeName'
+ARCHIVE = 'archiveCard'
+REOPEN = 'reopenCard'
+SET_DUE_DATE = 'setDueDate'
+CHANGE_DUE_DATE = 'changeDueDate'
+REMOVE_DUE_DATE = 'removeDueDate'
+ADD_LABEL = 'addLabelToCard'
+REMOVE_LABEL = 'removeLabelFromCard'
+ADD_MEMBER = 'addMemberToCard'
+REMOVE_MEMBER = 'removeMemberFromCard'
+ADD_ATTACHMENT = 'addAttachmentToCard'
+ADD_CHECKLIST = 'addChecklistToCard'
+COMMENT = 'commentCard'
+
+TRELLO_CARD_URL_TEMPLATE = '[{card_name}]({card_url})'
+
+ACTIONS_TO_MESSAGE_MAPPER = {
+    CREATE: 'created {card_url_template}',
+    CHANGE_LIST: 'moved {card_url_template} from {old_list} to {new_list}',
+    CHANGE_NAME: 'renamed the card from "{old_name}" to {card_url_template}',
+    ARCHIVE: 'archived {card_url_template}',
+    REOPEN: 'reopened {card_url_template}',
+    SET_DUE_DATE: 'set due date to {due_date} on {card_url_template}',
+    CHANGE_DUE_DATE: 'changed due date to {due_date} on {card_url_template}',
```

Comment:
Is it possible for us to retrieve the old date so we can say it like this? `changed due date for {card_url_template} from {old_due_date} to {due_date}`